test/ava-files.js

```diff
@@ -104,3 +104,29 @@ test('findFiles - does not return duplicates of the same file', function (t) {
 		t.end();
 	});
 });
+
+test('findFiles - finds the correct files by default', function (t) {
+	var avaFiles = new AvaFiles(['**/ava-files/default-patterns/**']);
```

Comment:
Sorry Kent, 
I see why you did it this way - It's exactly like the test I told you to copy. I'm updating your PR now. Expect a new PR and release soon.